llama_index/readers/schema/base.py

```diff
@@ -1,43 +1,50 @@
 """Base schema for readers."""
-from dataclasses import dataclass
 from typing import Optional
 
 from llama_index.bridge.langchain import Document as LCDocument
 
-from llama_index.schema import BaseDocument
+from llama_index.schema import TextNode, ObjectType
 
 
-@dataclass
-class Document(BaseDocument):
+class Document(TextNode):
```

Comment:
curious: should we move this under `llama_index.schema` as well?

Reply:
Since it depends on `TextNode` and is used in base index, seems like it's core enough to be there?

Reply:
Yea we can. That reminds me though, we will have to update the llama-hub schema as well